src/features/hud/components/InventoryItems.tsx

```diff
@@ -163,6 +168,16 @@ export const InventoryItems: React.FC<Props> = ({ onClose }) => {
                 className="h-12 w-12"
                 alt={selectedItem}
               />
+              {getCurrentTabIndex() === 0 && (
```

Comment:
```suggestion
              {currentTab === "Seeds" && (
```

One small suggestion, we can make this more readable using the `currentTab` variable